### ISSUE TITLE
rgbif CRAN check errors

### ISSUE THREAD

**Issue by jhnwllr** [ropensci/crul#179]
Hi @sckott 

I got a CRAN checks error email today. I believe it is likely `crul` related. 

I see that you recently fixed somethings related to CRAN checks. 
https://github.com/ropensci/crul/issues/178

```
Please see the problems shown on
<https://cran.r-project.org/web/checks/check_results_rgbif.html>.

Please correct before 2024-05-27 to safely retain your package on CRAN.
```

Do you have any advice here? 

My intuition is to update the minimum required version of crul to `v1.4.2`

**Comment by sckott** [comment]
Hi @jhnwllr - That fix in #178 was just for running tests - nothing to do with the package code. 

I think that HTTP framing layer issue is related to the server - so i imagine GBIF's servers - and ultimately comes from the package `curl` - see https://github.com/jeroen/curl/issues/156

**Comment by jhnwllr** [comment]
ok thank @sckott 

I investigated more and I think it is something else.
